packages/accessible-react-app/src/css/helpers.css

```diff
@@ -7,7 +7,7 @@
 button:focus,
 input:focus,
 textarea:focus {
-  outline: 2px solid #f9c642;
+  outline: 2px solid #face00;
```

Comment:
I am not familiar with this package `accessible-react-app`. It seems separate from Formation. Do we know where this "tool" is being used and if we need to create a release for it?

Reply:
Im not familiar with it either, this change is from doing a global search for the color value. Judging by the commit history it is not maintained. The only changes have been to the package.json file by dependabot. There is not even a name in the package.json, it is currently "simple-react-boilerplate". I would be surprised if this package is used anywhere.

Reply:
I think this was done as a proof of concept. I'm 99% confident it isn't being used anywhere.